lib/ansible/modules/files/lineinfile.py

```diff
@@ -94,6 +96,7 @@
       - If the first match is required, use C(firstmatch=yes).
       - A value is available; C(BOF) for inserting the line at the beginning of the file.
       - If specified regular expression has no matches, the line will be inserted at the end of the file.
+      - Cannot set with C(insertafter).
       - If regular expressions are passed to both C(regexp) and C(insertbefore), C(insertbefore) is only honored if no match for C(regexp) is found.
       - May not be used with C(backrefs).
```

Comment:
Combine line 99 with this line:
```suggestion
       - May not be used with C(backrefs) or C(insertafter).
```

Reply:
Thanks for reply. I fixed that. Could you check my change?